### ISSUE TITLE
Local Network: Issue enabling VLAN

### ISSUE THREAD

**Issue by unclelez** [seud0nym/tch-gui-unhide#152]
Hi,

When enabling VLAN in VLAN Filtering, It gives the error `invalid instance` when I try to save the config, it then triggers some issue with the DNS resolver, and within a few minutes, the DNS resolver stops working.

Any help would be great

Firmware: 20.3.c.0389-MR20-RA

![image](https://github.com/seud0nym/tch-gui-unhide/assets/112058383/d9cefe95-687a-48bc-830c-5d0829521256)

**Comment by seud0nym** [comment]
Sorry, I was sure I had answered this - must have closed the browser without hitting comment.

I have reproduced it, but I haven't tracked it down yet.

**Comment by seud0nym** [comment]
So I have found and fixed the problem, which then showed up some other problems. I know it was all working when I tested it originally, but I have obviously had a complete brain fade and subsequently changed something without testing properly. There were also issues with adding and deleting LAN interfaces and VLANs - things (especially in firewall configuration) weren't tidied up correctly.

My apologies, and I can only blame my advancing years...

There is a new release available now, which contains the fixes.

**Comment by unclelez** [comment]
Legend, thank you.

VLAN’s where the only feature that I though was lacking in this router.

Keep up the awesome work.

**Comment by gingerninja234** [comment]
Hi @seud0nym,

Sorry to hijack this issue, looks like I also have a problem when trying to set up VLANs.

I've just tried running VLANs from the Local Network screen however it disconnects my WAN connection straight away and switches over to the mobile backup. I have a PPPoE connection for my WAN

I also noticed that in the broadband card it shows "VLAN 1" as active. If I go into that card the VLAN settings have been switched off so it must be pulling it from the local network card (might be a clue to follow)

Big fan of you and what you've done so far with these modems, it's awesome! 

Thanks again for your help

Firmware Version: 20.3.c.0432-MR21.1-RA

<img width="1552" alt="Screen Shot 2023-06-02 at 5 37 37 pm" src="https://github.com/seud0nym/tch-gui-unhide/assets/22275247/59453450-2df2-4cf6-81cc-239c02871d7a">

<img width="1552" alt="Screen Shot 2023-06-02 at 5 37 27 pm" src="https://github.com/seud0nym/tch-gui-unhide/assets/22275247/37f287b9-b0a2-47b6-9750-843f22fc8b20">
